whitepaper/whitepaper.md

```diff
@@ -1553,7 +1597,12 @@ Web 3.0 applications (decentralized apps or *DApps*) which interact with blockch
 
 **Tx submitter**: The entity submits the remote transaction to the API server plug-in on one of the ledgers.
 
-**API Server**: A module of Hyperledger Cactus which provides a unified interface to control/monitor Blockchain ledger behind it.
+**API Server**: A Cactus package that is the backbone of the plugin architecture and the host component for all plugins with the ability to automaticaly wire up plugins that come with their own web services (REST or asynchronous APIs through HTTP or WebSockets for example)
```

Comment:
I do not want to define API Server as the CACTUS Node component since it is merely a common implementation of the server interface. We should name them with meaningful ones like "Validator Server", "CACTUS core API server".

Reply:
> I do not want to define API Server as the CACTUS Node component since it is merely a common implementation of the server interface. We should name them with meaningful ones like "Validator Server", "CACTUS core API server".

@sfuji822 In that notation, how would you refer to an API server that has plugins loaded for 3 or 4 different tasks (one of which could be validator)?